### DISCUSSION TITLE
How to "use" Polymorphic

### DISCUSSION THREAD

**Discussion by rodolphonetto** [adonisjs/core#1389] [Help]
Hy guys, I've a medias table that join with a lot of other tables, since Adonis/Lucid does not support polymorphism how do u solve it? create a table for every relationship? 

Could you share your ideias to help me solve this "issue"? 

Thanks ☺️

**Comment by exenestecnico** [comment]
I think this is related to my [question](https://github.com/adonisjs/core/discussions/1264).

**Comment by the-juju** [comment]
UP, any way to use polymorphic relations directly from Lucid ?
